storagenode/console/service.go

```diff
@@ -84,8 +86,11 @@ func NewService(log *zap.Logger, consoleDB DB, bandwidth bandwidth.DB, pieceStor
 
 	return &Service{
 		log:                log,
+		trust:              trust,
 		consoleDB:          consoleDB,
 		bandwidthDB:        bandwidth,
+		reputationDB:       reputationDB,
+		storageUsageDB:     storageUsageDB,
```

Comment:
I would like to see that we aggregate those two new DBs into a single group/struct

Reply:
https://github.com/storj/storj/pull/2742